docs/java-installation.md

```diff
@@ -103,19 +103,49 @@ Configure the [`Maven Surefire Plugin`](https://maven.apache.org/surefire/maven-
 </plugin>
 ```
 
+### Resolving source code filepaths
+
+Only necessary if your project has modules that have classes with the same name and package.
+
+Configure the `Metadata Maven Plugin` to attach the absolute source code filepath to every compiled class using an external file.
+
+```xml
+<plugin>
+  <groupId>com.undefinedlabs</groupId>
+  <artifactId>metadata-maven-plugin</artifactId>
+  <version>0.1.0-beta.1</version>
```

Comment:
are we going to use a beta?

Reply:
we shouldn't

Reply:
@drodriguezhdez at least we should have a `0.1.0` version without the beta tag